aws-tools/import_recent_dryad.sh

```diff
@@ -1,9 +1,18 @@
 #! /bin/bash
 
+# Usage: import_recent_dryad.sh [skipdb]
+#
+# If the 'skipdb' parameter is present, the database is not updated
 
 echo "grabbing latest from aws..."
-/home/ubuntu/.local/bin/aws s3 cp s3://dryad-backup/databaseBackups/dryadDBlatest.sql $HOME/dryadDBlatest.sql 
-import_pg_dump.sh $HOME/dryadDBlatest.sql
 
+if [ "a$1" != "askipdb" ]
```

Comment:
What is the reasoning for phrasing the conditional this way, with the prepended "a"?

Reply:
It's an old habit for ensuring that you don't get an error if the variable isn't defined. Looking at some examples, this probably is not necessary in bash.

Reply:
Can you change it?

Reply:
done